static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -177,23 +177,29 @@ class IssueAlertOptions extends DeprecatedAsyncComponent<Props, State> {
       </CustomizeAlert>,
     ];
 
-    const default_label = this.shouldUseNewDefaultSetting()
-      ? t('Alert me on high priority issues')
-      : t('Alert me on every new issue');
-
     const options: [string, React.ReactNode][] = [
-      [RuleAction.DEFAULT_ALERT.toString(), default_label],
-      ...(hasProperlyLoadedConditions ? [customizedAlertOption] : []),
       [RuleAction.CREATE_ALERT_LATER.toString(), t("I'll create my own alerts later")],
     ];
+
+    if (this.props.organization.features.includes('default-metric-alerts-new-projects')) {
+      options.unshift([RuleAction.DEFAULT_ALERT.toString(), t('Alert me on issues')]);
```

Comment:
I think "Alert me on issues" still implies issue alerts only, maybe "Alert me on issues and metrics" or something vague like "Alert me when things are bad" idk I am not a copywriter 😅